lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -273,6 +274,7 @@ def upload_ora2_data(
 
     try:
         header, datarows = OraAggregateData.collect_ora2_data(course_id)
+        header, datarows = generate_custom_ora2_report(header, datarows)  # Added for LUMS custom requirements.
```

Comment:
The above line generates the CSV file it can't be removed we are updating the previously generated csv file here as you can see we are passing the same parameters to the function.